modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1825,6 +1825,561 @@
       ],
       "description": "Processes the nested list of message processors asynchronously using a thread pool",
       "modelProperties": {}
+    },
+    {
+      "name": "try",
+      "kind": "scope",
+      "blocking": true,
+      "executionType": "CPU_LITE",
+      "errors": [],
+      "output": {
+        "type": {
+          "format": "java",
+          "type": "Any"
+        },
+        "hasDynamicType": false,
+        "description": "",
+        "modelProperties": {}
+      },
+      "outputAttributes": {
+        "type": {
+          "format": "java",
+          "type": "Object",
+          "annotations": {
+            "typeId": "org.mule.runtime.api.message.Attributes",
+            "classInformation": {
+              "hasDefaultConstructor": false,
+              "isInterface": true,
+              "isInstantiable": false,
+              "isAbstract": true,
+              "isFinal": false,
+              "implementedInterfaces": [
+                "java.io.Serializable"
+              ],
+              "parent": "",
+              "genericTypes": []
+            },
+            "typeAlias": {
+              "value": "Attributes"
+            }
+          },
+          "fields": []
+        },
+        "hasDynamicType": false,
+        "description": "",
+        "modelProperties": {}
+      },
+      "transactional": true,
+      "requiresConnection": false,
+      "parameterGroupModels": [
+        {
+          "parameters": [
+            {
+              "type": {
```

Comment:
this is the `transactionalAction` model that's generated for a transactional operation in DB:
`{
                  "type": {
                    "format": "java",
                    "type": "String",
                    "annotations": {
                      "typeId": "org.mule.runtime.extension.api.tx.OperationTransactionalAction",
                      "enum": {
                        "type": "[Ljava.lang.String;",
                        "values": [
                          "ALWAYS_JOIN",
                          "JOIN_IF_POSSIBLE",
                          "NOT_SUPPORTED"
                        ]
                      }
                    }
                  },
                  "hasDynamicType": false,
                  "required": false,
                  "expressionSupport": "NOT_SUPPORTED",
                  "defaultValue": "JOIN_IF_POSSIBLE",
                  "role": "BEHAVIOUR",
                  "dslConfiguration": {
                    "allowsInlineDefinition": true,
                    "allowsReferences": true,
                    "allowTopLevelDefinition": false
                  },
                  "layoutModel": {
                    "password": false,
                    "text": false,
                    "query": false,
                    "tabName": "Transactions"
                  },
                  "name": "transactionalAction",
                  "description": "The type of joining action that operations can take regarding transactions.",
                  "modelProperties": {}
                }`